shuffle/cli/src/transactions.rs

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+use crate::shared::{get_home_path, Home};
+use anyhow::{anyhow, Result};
+use diem_types::account_address::AccountAddress;
+use reqwest::{Client, Response};
+use serde_json::Value;
+use std::{cmp::max, fs, io, io::Write, str::FromStr, thread, time};
+use url::Url;
+
+// Will list the last 10 transactions and has the ability to block and stream future transactions.
+pub async fn handle(network: Url, tailing: bool) -> Result<()> {
+    let home = Home::new(get_home_path().as_path())?;
+    let address_str = fs::read_to_string(home.get_latest_address_path())?;
+    let address = AccountAddress::from_str(address_str.as_str())?;
+    let client = reqwest::Client::new();
+
+    let path = network.join(format!("accounts/{}/transactions", address).as_str())?;
+    let seq_num = get_sequence_number(&client, &network, address).await?;
+    let start = max(seq_num as i64 - 10, 0);
+
+    let resp = client
+        .get(path.as_str())
+        .query(&[("start", start.to_string().as_str())])
+        .query(&[("limit", "10")])
+        .send()
+        .await?;
+
+    let json: serde_json::Value = serde_json::from_str(resp.text().await?.as_str())?;
+
+    let all_transactions = json
+        .as_array()
+        .ok_or_else(|| anyhow::anyhow!("Failed to get transactions"))?;
+
+    for txn_index in 0..all_transactions.len() {
+        write_into(&mut io::stdout(), &json[txn_index])?;
+        println!();
+        println!();
+    }
+
+    let mut old_seq_num = 0;
+    if !all_transactions.is_empty() {
+        old_seq_num = all_transactions
+            .get(all_transactions.len() - 1)
+            .ok_or_else(|| anyhow!("Couldn't get last transaction"))?["sequence_number"]
+            .to_string()
+            .replace('"', "")
+            .parse::<u64>()?;
```

Comment:
I'm curious what string ends up being parsed after the `.replace('"',...)`. Let's pull this out into a separate method and test it.

I imagine
```rust
old_seq_num = parse_txn_seq_number(all_transactions.last);
```

https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.last